### PR TITLE
[RHDX-456] Feature/web components not supported in ie11 rhdx 4560

### PR DESCRIPTION
**Pull request by alistairmc**
### JIRA Issue Link
https://projects.engineering.redhat.com/browse/RHDX-456

### Verification Process
Check site functions in ie11

### CODE REVIEW

**Comment by alistairmc** [comment]
retest this please

**Comment by alistairmc** [comment]
retest this please

**Comment by alistairmc** [comment]
rebuild this please

**Comment by alistairmc** [comment]
retest this please

**Comment by alistairmc** [comment]
retest this please

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/0-test-for-css-var.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        link_override.rel = 'stylesheet';  
+        link_override.type = 'text/css'; 
+        link_override.href = '/themes/custom/rhdp2/css/rhd.ie11-overrides.css';  
+  
+        // Append link elements to HTML head
+        head.appendChild(link);  
+        head.appendChild(link_override);  
+
+        // Remove the non ie stylesheet
+        $('link[href^="/themes/custom/rhdp2/rhd-frontend/dist/css/rhd.min.css"]').prop('disabled', true).remove();
```
Comment (staceymosier) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/0-test-for-css-var.js:27]: @alistairmc  I don't think this link exists in pr/prod, where css aggregation is enabled. But I don't think it changes the effect in IE11 by now.

Comment (alistairmc) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/0-test-for-css-var.js:27]: It should not throw an error if the sector is not matched nothing will happen

**Pull request merged**